keylime/tenant_webapp.py

```diff
@@ -335,135 +335,135 @@ def get(self):
+            agents[agent] = self.get_agent_state(agent_id)
+
+        # Pre-create sorted agents list
         sorted_by_state = {}
-        states = cloud_verifier_common.CloudInstance_Operational_State.STR_MAPPINGS
+        states = cloud_verifier_common.CloudAgent_Operational_State.STR_MAPPINGS
         for state in states:
             sorted_by_state[state] = {}
-        
-        # Build sorted instances list 
-        for instance_id in instances:
-            state = instances[instance_id]["operational_state"]
-            sorted_by_state[state][instance_id] = instances[instance_id]
-        
+
+        # Build sorted agents list
+        for agent_id in agents:
+            state = agents[agent_id]["operational_state"]
+            sorted_by_state[state][agent_id] = agents[agent_id]
+
         print_order = [10,9,7,3,4,5,6,2,1,8,0]
-        sorted_instances = []
+        sorted_agents = []
         for state in print_order:
-            for instance_id in sorted_by_state[state]:
-                sorted_instances.append(instance_id)
-        
-        common.echo_json_response(self, 200, "Success", {'uuids':sorted_instances})
+            for agent_id in sorted_by_state[state]:
+                sorted_agents.append(agent_id)
+
+        common.echo_json_response(self, 200, "Success", {'uuids':sorted_agents})
 
     def delete(self):
-        """This method handles the DELETE requests to remove instances from the Cloud Verifier. 
-         
-        Currently, only instances resources are available for DELETEing, i.e. /nodes. All other DELETE uri's will return errors.
-        instances requests require a single instance_id parameter which identifies the instance to be deleted.    
+        """This method handles the DELETE requests to remove agents from the Cloud Verifier.
+
+        Currently, only agents resources are available for DELETEing, i.e. /agents. All other DELETE uri's will return errors.
+        agents requests require a single agent_id parameter which identifies the agent to be deleted.
         """
-        
+
         rest_params = common.get_restful_params(self.request.uri)
         if rest_params is None:
-            common.echo_json_response(self, 405, "Not Implemented: Use /nodes/ interface")
+            common.echo_json_response(self, 405, "Not Implemented: Use /agents/ interface")
             return
-        
-        if "nodes" not in rest_params:
+
+        if "agents" not in rest_params:
             common.echo_json_response(self, 400, "uri not supported")
             logger.warning('DELETE returning 400 response. uri not supported: ' + self.request.path)
             return
-        
-        instance_id = rest_params["nodes"]
-        
-        # let Tenant do dirty work of deleting node 
+
+        agent_id = rest_params["agents"]
+
+        # let Tenant do dirty work of deleting agent
         mytenant = tenant.Tenant()
-        mytenant.node_uuid = instance_id
+        mytenant.agent_uuid = agent_id
```

Comment:
Does this still work?  I thought this variable was changed to `AGENT_UUID` in tenant.py (uppercase now)?